kale/embed/uncertainty_fitting.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import os
+from typing import List, Optional, Tuple
+
+import numpy as np
+import pandas as pd
+from yacs.config import CfgNode
+
+from kale.interpret.uncertainty_quantiles import quantile_binning_and_est_errors
+from kale.loaddata.tabular_access import load_csv_columns
+from kale.predict.uncertainty_binning import quantile_binning_predictions
+from kale.prepdata.tabular_transform import apply_confidence_inversion
+
+
+def fit_and_predict(
+    landmark: int,
```

Comment:
Does this only apply to landmarks? QBin should be generic. If it is not a landmark, it should be still applicable, right?

Reply:
Potential Improvement: change all instances of "landmark" to "target_indexes"